emscripten.py

```diff
@@ -550,6 +550,13 @@ def finalize_wasm(infile, outfile, memfile, js_syms):
   metadata = get_metadata(outfile, outfile, modify_wasm, args)
   if modify_wasm:
     building.save_intermediate(outfile, 'post_finalize.wasm')
+  else:
+    # wasm-emscripten-finalize will strip the features section for us
+    # automatically, but if we did not modify the wasm then we didn't run it,
+    # and must strip it manually. note that we must do this *after* getting the
+    # metadata, as the features are among the metadata we collect.
+    building.save_intermediate(outfile, 'strip-features.wasm')
+    building.strip(infile, outfile, sections=['target_features'])
```

Comment:
I assume this uses some other tool to very efficiently strip the section?

Reply:
Yes, `llvm-objdump`

Reply:
Can we rely on `wasm-opt` to do this?  That runs in all the modes where we care about stripping it right?

Reply:
(hopefully that will be quicker than launching an extra sub-process just for this).

Reply:
Currently we want to always strip it, even with `wasm-opt` is not run (and the metadce-minimal tests test that). But we could perhaps decide to stop caring in `-O0` and `-O1`? Let's think about that separately.

Reply:
I think is we decide not to run wasm-opt we should probably also decide not to strip this section.  It seems like if you are building for production then wasm-opt will be run anyway, right?